api/app/Service/Forms/FormLogicConditionChecker.php

```diff
@@ -348,6 +392,10 @@ private function textConditionMet(array $propertyCondition, $value): bool
                 } catch (\Exception $e) {
                     return true;
                 }
+            case 'exists_in_submissions':
```

Comment:
I know we talked about this but I don't understand why we have to negate this here? For other conditions we don't negate so there must be an issue it doens't make sense...?